lib/benchee/formatters/unit.ex

```diff
@@ -0,0 +1,118 @@
+defmodule Benchee.Unit do
+  @moduledoc """
+  Provides scaling and labeling functions for numbers. Use "count" functions
+  for numbers that represent countable things, such as iterations per second.
+  Use "duration" functions for numbers that represent durations, such as run
+  times.
+  """
+
+  @type unit :: atom
+  @type scaled_number :: {number, unit}
+
+  # In 1.3, this could be declared as `keyword`, but use a custom type so it
+  # will also compile in 1.2
+  @type options ::[{atom, atom}]
```

Comment:
Yeah it was kind of a bummer to realize that I had to undo the `describe`s (I forgot those wouldn't work on 1.2), but there were only 3, and the tests aren't actually that much harder to read.